get_imports.py

```diff
@@ -173,11 +174,23 @@ def get_libs_for_project(project_name):
         if file.endswith(".py"):
 
             found_imports = findimports.find_imports(f"{project_dir}{file}")
-
             for cur_import in found_imports:
                 cur_lib = cur_import.name.split(".")[0]
                 if cur_lib in bundle_data or cur_lib in community_bundle_data:
                     found_libs.add(cur_lib)
+                if cur_import.name.endswith(".*"):
+                    filepath = os.path.join(
+                        project_dir, "/".join(cur_import.name[:-2].split(".")) + ".py"
```

Comment:
Use `os.path.join(` instead of `"/".join(`

Reply:
changed to os.path.join in the latest commits.